src/bridge-canister/src/bridge.rs

```diff
@@ -135,6 +135,7 @@ pub enum OperationAction<Op> {
     Create(Op),
     CreateWithId(OperationId, Op),
     Update { nonce: u32, update_to: Op },
+    CreateWithIdAndMemo(OperationId, Op, String),
```

Comment:
It would be more ergonomic to add `Option<Memo>` parameter to the `Create` and `CreateWithId` variants, that to add a separate variant.